languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -44473,7 +44476,7 @@ USA
                     <exception case_sensitive="yes">A</exception>
                 </token>
             </pattern>
-            <message>The verb "\8" seems to be incorrect here. Did you mean <suggestion><match no="8" postag="VBN" /></suggestion>, <suggestion>been <match no="8" postag="VBG" /></suggestion> or <suggestion>to <match no="8" postag="VB" /></suggestion>?</message>
+            <message>The verb "\8" seems to be incorrect here. Did you mean <suggestion><match no="8" postag="VBN" /></suggestion>, <suggestion>been <match no="8" postag="VBG" /></suggestion>, or <suggestion>to <match no="8" postag="VB" /></suggestion>?</message>
```

Comment:
This is another case of the first sentence being enough. The suggestions could be taken out of the message.